src/digitalmarketplace/components/list-input/template.njk

```diff
@@ -58,7 +58,11 @@
   <div class="dm-list-input__item-container" role="region" aria-live="assertive">
     {% for i in range(0, params.maxItems) -%}
       {%- set item = params.items[i] if params.items[i] else {} -%}
-      {%- set entryItemId = params.id ~ '-' ~ loop.index -%}
+      {%- if loop.first -%}
```

Comment:
👍 

Suspect this may impact Briefs, since presumably we worked around this missed functionality.

Reply:
Ah, well it turns out in briefs currently the error summary link doesn't work 😅

Reply:
😱